components/tools/OmeroWeb/test/integration/test_tree.py

```diff
@@ -1782,9 +1782,9 @@ def test_marshal_images_thumb_version(self, userA, image_pixels_userA):
         Test marshalling image, loading thumbnail version
         """
         conn = get_connection(userA)
-        # Thumbnail not yet created: version will be -1
+        # Thumbnail is created, see testlib.ITest.createTestImage
         expected = expected_images(userA, [image_pixels_userA],
-                                   extraValues={'thumbVersion': -1})
+                                   extraValues={'thumbVersion': 0})
```

Comment:
I agree that we should test if view gives correct version on pyramid creation, but `test_marshal_images_thumb_version` never done it before, so if we need more extensive testing they have to be written in line with https://github.com/openmicroscopy/openmicroscopy/blob/metadata52/components/tools/OmeroPy/test/integration/test_thumbs.py.

I can introduce `with_thumb` flag to `testlib.ITest.createTestImage` and parametrize existing test.